Tasks/VsTestV2/make.json

```diff
@@ -6,7 +6,7 @@
                 "dest": "./"
             },
             {
```

Comment:
we need to port to release branch as well

Reply:
yes